windeploy_helper.sh

```diff
@@ -33,10 +33,9 @@ elif [[ "$platform" == "mingw32" ]]; then
 	PLATFORM_FILES="libgcc_s_dw2-1.dll" 
 fi
 
-for f in $FILES; do cp $MSYSTEM_PREFIX/bin/$f $TARGET; done
+for f in "${FILES[@]}"; do cp $MSYSTEM_PREFIX/bin/$f $TARGET || exit 1; done
 
-for f in $ICU_FILES; do cp $MSYSTEM_PREFIX/bin/$f $TARGET; done
-
-for f in $PLATFORM_FILES; do cp $MSYSTEM_PREFIX/bin/$f $TARGET; done
+for f in ${ICU_FILES[@]}; do cp $MSYSTEM_PREFIX/bin/$f $TARGET || exit 1; done
```

Comment:
Best to also double-quote `${ICU_FILES[@]}` here and `${PLATFORM_FILES[@]}` below.
This will prevent nonsense if there is ever spaces or shebang chars in a filename.

Reply:
Thanks, pushed.